### PR TITLE
Backport lost tinymce translations

### PR DESCRIPTION
**Pull request by djowett-ftw**
There might be more work to do here, but I would like to backport this translations fix
 
However ... I need pointers of what to do next

### CODE REVIEW

**Comment by djowett-ftw** [comment]
@jenkins-plone-org please run jobs

**Comment by djowett-ftw** [comment]
@jenkins-plone-org please run jobs

File: mockup/patterns/tinymce/js/links.js [hunk 1]
```diff
@@ -466,6 +466,14 @@ define([
         upload: this.options.upload,
         text: this.options.text,
         insertHeading: this.options.text.insertHeading,
+        insertImageHelp: this.options.text.insertImageHelp,
+        ExternalImage: this.options.text.ExternalImage,
```
Comment (thet) [inline mockup/patterns/tinymce/js/links.js]: is ``externalImage`` in mockup master

Comment (thet) [reply mockup/patterns/tinymce/js/links.js]: there are also other upper/lowercase inconsitencies here.
@djowett-ftw could you please just copy over the whole generateModalHtml function from master? I'd also like to have the attributes in the same order.

**Comment by thet** [comment]
@djowett-ftw please fix the upper/lower case issues and update the templates as well.
For which Plone version do you need these changes?

After this gets merged you need to create a new bundle for plone-logged-in for CMFPlone (no staticresources available for Plone < 5.2) with ./bin/plone-compile-resources

File: mockup/patterns/tinymce/js/links.js (hunk 1 above)
Comment (djowett-ftw) [reply mockup/patterns/tinymce/js/links.js]: Hi @thet - good idea I've done that.

**Comment by djowett-ftw** [comment]
> @djowett-ftw please fix the upper/lower case issues and update the templates as well.
> For which Plone version do you need these changes?

Hi @thet - thanks for the guidance, sorry it took a while.   I think this is done now, as you requested.  We are looking for this to work with Plone 5.1 (we're currently on 5.1.6).

There are some missing German translations I'll put in plone.app.locales which depend on this too.

> After this gets merged you need to create a new bundle for plone-logged-in for CMFPlone (no staticresources available for Plone < 5.2) with ./bin/plone-compile-resources

Understood

**Comment by djowett-ftw** [comment]
@jenkins-plone-org please run jobs

**Comment by djowett-ftw** [comment]
@thet I think this is correct now - with https://github.com/collective/plone.app.locales/pull/287
But the test above hasn't reported back even though it appears to be finished.

**Comment by djowett-ftw** [comment]
Trying to close / open to get Jenkins to restart

**Comment by thet** [comment]
@djowett-ftw tnx for taking care of this!

**Pull request merged**